src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -49,6 +49,8 @@
 import org.bukkit.util.Vector;
 
 import io.github.thebusybiscuit.cscorelib2.materials.MaterialTools;
+import io.github.thebusybiscuit.cscorelib2.protection.ProtectionManager;
+import io.github.thebusybiscuit.cscorelib2.protection.ProtectionModule;

```

Comment:
Careful! You also need to update the maven-shade-plugin configuration in the pom.xml to access these at runtime!